documentation/tutorials/multisite-workflow/site_info_example.csv

```diff
@@ -0,0 +1,11 @@
+id,lat,lon,elev,site_group
```

Comment:
As commented above, I recommend deleting this file -- we show the whole contents inline in enough detail for users to construct their own.